contrib/test_jenkins.sh

```
@@ -612,7 +612,9 @@ build_jucx() {
 #
 build_armclang() {
 	echo 1..1 > build_armclang.tap
-	if module_load arm-compiler/latest
+	armclang_test_file=$(mktemp ./XXXXXX).c
```

@yosefe my concern that you will end up silently disabling compilers without any notification. We will end up working under the assumption that everything tested, while nothing actually happens

---

@shamisp what would you suggest?

---

@yosefe just as it was before - fail. it will raise the alert in CI and then people go and fix relevant issues.

---

is it possible to get long term license of armclang compiler for UCX testing?

---

@yosefe 1 year is a long term :) 
We just need setup a reminder in place. The license issue was solved. Can you please check that it work for you ? I would strongly advice to remove this patch since it will results in silent failure that will bite us later on.

---

thanks @shamisp , i'm ok with reverting this patch.